src/SDKs/DataLake.Analytics/Management.DataLake.Analytics/Customizations/DataLakeAnalyticsCustomizationHelper.cs

```diff
@@ -14,7 +14,7 @@ internal static class DataLakeAnalyticsCustomizationHelper
         /// This constant is used as the default package version to place in the user agent.
         /// It should mirror the package version in the project.json file.
         /// </summary>
-        internal const string PackageVersion = "3.1.1-preview";
+        internal const string PackageVersion = "3.1.2-preview";
```

Comment:
@ro-joowan can you explain what is this package number and what project.json file are you referring to.

Reply:
@shahabhijeet Couldn't find a project.json file in the src.  Ben might be referring to the csproj file (https://github.com/ro-joowan/azure-sdk-for-net/blob/341b0bbea8d9110f2047f9d28150a4cc95ef34fc/src/SDKs/DataLake.Analytics/Management.DataLake.Analytics/Microsoft.Azure.Management.DataLake.Analytics.csproj), as this is one of the three places I've been told to update the version number -- the other place being in AssemblyInfo.cs

As for its purpose, I found this (https://github.com/ro-joowan/azure-sdk-for-net/blob/341b0bbea8d9110f2047f9d28150a4cc95ef34fc/src/SDKs/DataLake.Analytics/Management.DataLake.Analytics/Customizations/DataLakeAnalyticsAccountManagementClient.Customizations.cs):

/// <param name='userAgentAssemblyVersion'>
/// Optional. The version string that should be sent in the user-agent header for all requests. The default is the current version of the SDK.
/// </param>

It looks like this custom method is a way for the user to set the SDK version.

Reply:
@shahabhijeet And so this PackageVersion is used as the default SDK version in the event that the user does not provide a custom version (https://github.com/ro-joowan/azure-sdk-for-net/blob/psSdkJson6/src/SDKs/DataLake.Analytics/Management.DataLake.Analytics/Customizations/DataLakeAnalyticsCustomizationHelper.cs#L29)

Reply:
Let's revisit this in the next release.
Will allow now.